dls-controls/malcolmjs
# Bugfix/port linking to self #234

## Description

Fixed a bug in the context menu deleting a link.
Fixed a bug where a port could link to itself.
Added some more tests for the layout related code.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Run the site, try to right click a link twice and check it doesn't delete. Click on a port and check there isn't a send in the redux messages in the console.

## Agile board tracking

connect to #234
closes #234
